examples/nodejs-simple/Makefile

```diff
@@ -43,7 +43,7 @@ build:
 	cd $(root_path) && docker build -f $(project_path)/Dockerfile --tag=$(server_tag) .
 
 run:
-	docker run --network=host $(server_tag)
+	docker run --network=host $(server_tag) $(args)
```

Comment:
Nice stuff - will be useful for testing #1299 :+1: 

We should also increment the tag version in this file, so we can rebuild and push it up for people to use.

Reply:
Would that be server-tag and should it be 1.5 now?

Reply:
`server_tag = $(REPOSITORY)/nodejs-simple-server:0.3` :arrow_right: `server_tag = $(REPOSITORY)/nodejs-simple-server:0.4`